### PR TITLE
sync labels to claims

### PR DESCRIPTION
**Pull request by zhiweiyin318**
story: https://github.com/open-cluster-management/backlog/issues/11779

### CODE REVIEW

File: pkg/klusterlet/clusterclaim/clusterclaim_controller.go [hunk 1]
```diff
@@ -46,11 +49,34 @@ func (r *ClusterClaimReconciler) syncClaims(ctx context.Context) error {
...
+		claimSet.Insert(claim.Name)
+	}
+
+	existedObjs, err := r.ClusterClient.ClusterV1alpha1().ClusterClaims().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		errs = append(errs, client.IgnoreNotFound(err))
+		return utils.NewMultiLineAggregate(errs)
+	}
+
+	for _, claim := range existedObjs.Items {
```
Comment (qiujian16) [inline pkg/klusterlet/clusterclaim/clusterclaim_controller.go:68]: this seems unnecessary. Why not 

```go
for _, claim := range existedObjs.Items {
  if claimSet.Has(claim.Name) {
    continue
  }

  
  err := r.ClusterClient.ClusterV1alpha1().ClusterClaims().Delete(ctx, claim, metav1.DeleteOptions{})
  if err != nil {
    errs = append(errs, err)
   }
}
```

Comment (zhiweiyin318) [reply pkg/klusterlet/clusterclaim/clusterclaim_controller.go:67]: done

**Review by qiujian16 — COMMENTED** [review]
/lgtm

**Pull request merged**